raiden/raiden_service.py

```diff
@@ -317,20 +317,16 @@ def __init__(
             BLOCK_ID_LATEST
         )
         invalid_settle_timeout = (
-            config.settle_timeout < settlement_timeout_min
-            or config.settle_timeout > settlement_timeout_max
+            settlement_timeout_max > config.settle_timeout < settlement_timeout_min
```

Comment:
Did you accidentally turn the `or` into and `and` while refactoring?

Reply:
Yes. I was going to switch this into `valid_settle_timeout`, so I could check an interval, but decided was not worth it, and mistakenly left the `and` instead of an `or` 😳 

fixed ✅